core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2346,6 +2308,72 @@ private[spark] class DAGScheduler(
     }
   }
 
+  private def collectSucceedingStages(mapStage: ShuffleMapStage): HashSet[Stage] = {
```

Comment:
I assume these methods just moved code from https://github.com/apache/spark/pull/50630/files#diff-85de35b2e85646ed499c545a3be1cd3ffd525a88aae835a9c621f877eebadcb6L2132 ?

Reply:
Yes, you are right!